verification/cbmc/proofs/Makefile-project-defines

```diff
@@ -39,8 +39,7 @@ DEFINES += -DAWS_DEEP_CHECKS=$(AWS_DEEP_CHECKS)
 
 
 # Extra CBMC flags not enabled by Makefile.common
-CHECKFLAGS += --enum-range-check
-CHECKFLAGS += --pointer-primitive-check
```

Comment:
I request that @danielsn approves this line before we merge this.

Reply:
The changes I made in this PR fix all the enum-range-check issues as well, so we don't _need_ to disable it, but @feliperodri suggested that we do

Reply:
I don't mind about --enum-range-check, but I'm very concerned about disabling --pointer-primitive-check.

Reply:
So that is a duplicate. The next line also adds `--pointer-primitive-check`.

Reply:
He is not disabling it, there was an extra line. @karkhaz

Reply:
Excuse me, I misread. Thank you!